src/main/java/org/team1540/bunnybotTank2023/commands/drivetrain/Drivetrain.java

```diff
@@ -60,19 +44,24 @@ public Drivetrain(AHRS gyro) {
 
     @Override
     public void periodic() {
+        io.updateInputs(inputs);
+        Logger.getInstance().processInputs("Drivetrain", inputs);
+
         DifferentialDriveWheelPositions wheelPositions = getWheelPositions();
         poseEstimator.update(getYaw(), wheelPositions.leftDistanceMeters, wheelPositions.rightDistanceMeters);
+        Logger.getInstance().recordOutput("odometry/robotPose", getPose());
     }
 
     //Speed between -1 and 1
     public void drive(double leftInput, double rightInput) {
-        frontLeft.set(leftInput);
-        frontRight.set(rightInput);
+        io.setVoltage(
+                leftInput * 12.0,
+                rightInput * 12.0
```

Comment:
Is this the way PercentOutput converts to voltage or do they do some kind of calculation on the actual input voltage?

Reply:
Voltage compensation is enabled, which allows this to have the same behavior as PercentOutput